tests/tck/features/yield/parameter.feature

```diff
@@ -7,16 +7,48 @@ Feature: Parameter
     Given a graph with space named "nba"
     Given parameters: {"p1":1,"p2":true,"p3":"Tim Duncan","p4":3.3,"p5":[1,true,3],"p6":{"a":3,"b":false,"c":"Tim Duncan"},"p7":{"a":{"b":{"c":"Tim Duncan","d":[1,2,3,true,"Tim Duncan"]}}},"p8":"Manu Ginobili", "p9":["Tim Duncan","Tony Parker"]}
 
-  Scenario: return parameters
+  Scenario: [param-test-001] without define param
```

Comment:
Add some test scenarios where parameter expressions and aggregate expressions are mixed.

Reply:
done